scripts/ts/checkOutdatedDependencies/checkOutdatedDependencies.ts

```diff
@@ -12,6 +12,9 @@ import { DependenciesTable } from "./types/DependeciesTable";
 import { getSeverityType } from "./types/GroupBySeverity";
 import { getDependencyType, increaseSeverityAmount } from "./types/GroupByType";
 import { OutdatedStats, toOutdatedPackage } from "./types/OutdatedStats";
+import { pipe } from "fp-ts2/lib/function";
+
+import { eitherToV2 } from "../../../ts/migration/fp-ts-converters";
```

Comment:
When needed we can use converters v1 <-> v2.

You can remove the conversion after the code you depend on is migrated to v2. Better explained below.